api/types/src/move_types.rs

```diff
@@ -226,6 +226,29 @@ impl TryFrom<AnnotatedMoveStruct> for MoveStructValue {
 
     fn try_from(s: AnnotatedMoveStruct) -> anyhow::Result<Self> {
```

Comment:
Is it OK to keep option as it is in AnnotatedMoveStruct and translate into JSON format explicitly?